products-service/lib/products-service-stack.ts

```diff
@@ -71,13 +102,41 @@ export class ProductsServiceStackKate extends cdk.Stack {
       ...lambdaFunctionProps,
     });
 
+    const catalogBatchProcess = new LambdaFunction(
+      this,
+      "CatalogBatchProcessHandlerKate",
+      {
+        handler: "catalogBatchProcess.handler",
+        ...lambdaFunctionProps,
+      }
+    );
+
+    const deadLetterQueue = new Queue(this, "DeadLetterQueueKate");
```

Comment:
Could you please elaborate how dead letter queue is used?

Reply:
oh it's not really needed here, just wanted to try it:)